home/ivp-planning/advanced-tools/fence-tool.md

```diff
@@ -40,7 +40,7 @@ Pass-troughs have a flexible width, they always span from the selected point to
 
 ## Fence types and materials:
 
-It is possible to change the type respectively material of the fence. To do so, **choose an option from the list** available in the [path panel](../user-interface/path-panel.md).
+It is possible to change the type respectively material of the fence. To do so, **choose an option from the list**.
```

Comment:
While we're at it: Can we change "It is possible to change the type respectively material of the fence." to just "It is possible to change the material of the fence."

Reply:
changed.